shared-data/contrib/hints/manifest.json

```diff
@@ -15,7 +15,8 @@
         "/page/hints/deletion.html": {"file": "hints/deletion.html"},
         "/page/hints/backups.html": {"file": "hints/backups.html"},
         "/page/hints/gravatar.html": {"file": "hints/gravatar.html"},
-        "/page/hints/spam.html": {"file": "hints/spam.html"},
+        "/page/hints/organize-sidebar.html": {"file": "hints/organize-sidebar.html"},
+	"/page/hints/spam.html": {"file": "hints/spam.html"},
```

Comment:
Did the indentation change here? Or are you using TAB characters instead of spaces?